### DISCUSSION TITLE
[@types/intercom-web] IntercomCommandSignature is missing `showConversation`

### DISCUSSION THREAD

**Discussion by jenniferyhwu** [DefinitelyTyped/DefinitelyTyped#67520] [Issues with a @types package]
This method is supported by Intercom and documented [here](https://developers.intercom.com/installing-intercom/web/methods/#intercomshowconversation-conversationid), but isn't a possible value for the interface `IntercomCommandSignature`. There might be more methods missing from the type, but this is the one I need 🙏 

I think this would be a one-line change [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0904159479fa886e08cf323ec4d15dadb44355b4/types/intercom-web/index.d.ts#L46), happy to submit a PR for this if needed, please lmk!

[a comment by an automated account is not shown]

**Comment by teol99** [comment]
Stumbled across this while working on something that required the same type. Created and merged a PR [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69362)
